sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/private_endpoint.py

```diff
@@ -22,6 +36,14 @@ def __init__(
 
 
 class PrivateEndpoint:
+    """Private Endpoint of a workspace.
+
+    :param approval_type: Approval type of the private endpoint.
```

Comment:
Do we have a list of accepted approval values we could add here?

Reply:
This one I am not so sure on. For one, I do not see how these PrivateEndpoint or EndpointConnection objects are ever being used anywhere in the code. Wondering if these entities were left over some previous uncompleted work or something. let me try and get more information on this

Reply:
No worries. If we don't know immediately, let's just make a work item for it and address it later on. It's not a blocker.